src/test/java/com/github/romankh3/image/comparison/ImageComparisonUtilUnitTest.java

```diff
@@ -9,36 +9,37 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-import java.awt.*;
+import java.awt.Frame;
+import java.awt.Image;
 import java.awt.image.BufferedImage;
 import java.io.File;
-import java.io.FileNotFoundException;
 import java.io.IOException;
 import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * Unit-level testing for {@link ImageComparisonUtil} object.
  */
-public class ImageComparisonUtilUnitTest {
+public class ImageComparisonUtilUnitTest extends BaseTest {
 
     @Test
     @Ignore
-    public void testFrameMethod() throws IOException {
+    public void testFrameMethod() {
         BufferedImage image = readImageFromResources("result.png");
         Frame resultFrame = createGUI(image);
         assertEquals(image.getHeight(), resultFrame.getHeight());
         assertEquals(image.getWidth(), resultFrame.getWidth());
     }
 
-    @Test(expected = IOException.class)
-    public void testWrongPath() throws IOException {
+    @Test
+    public void testWrongPath() {
         //when-then
-        readImageFromResources("wrong-file-name.png");
+        Throwable ex = getException(() -> readImageFromResources("wrong-file-name.png"));
+        assertTrue(ex.getMessage().startsWith("Image wrong-file-name.png not found"));
```

Comment:
That's a bad idea to verify that the exception has a particular message. If you see such cases, you're probably doing smth wrong, e.g. having a single uber-exception.

In some cases it may be actually OK to do it, but, if it is so, at least to it in a way the framework proposes to you.